documentation/modules/ROOT/pages/index.adoc

```diff
@@ -27,39 +22,38 @@ xref:samm-specification:ROOT:index.adoc[Specification]
 [.icon-cli]
 --
 [.title]
-SAMM Command Line Interface (SAMM-CLI)
+Aspect Model Editor
 
 [.text]
-Command line interface to work with Aspect Models: Validate and transform models into documentation
-and code.
+A visual editor for your Aspect Models. Validates your models automatically against the SAMM specification.
 
 [.link]
-xref:esmf-developer-guide:tooling-guide:samm-cli.adoc[SAMM CLI]
+xref:ame-guide:ROOT:introduction.adoc[Aspect Model Editor]
 --
 
 [.tile]
 [.icon-cli]
 --
 [.title]
-Aspect Model Editor
+SAMM Command Line Interface +
+(SAMM CLI)
 
 [.text]
-The Aspect Model Editor is a visual editor for Aspect Models, which automatically validates the
-models against the BAMM Aspect Meta Model version.
+A CLI to validate your Aspect Models against SAMM and transform them to documentation and code.
 
 [.link]
-xref:ame-guide:ROOT:introduction.adoc[Aspect Model Editor]
+xref:esmf-developer-guide:tooling-guide:samm-cli.adoc[SAMM CLI]
 --
 
 [.tile]
 [.icon-cli]
 --
 [.title]
-Java ESMF SDK
+Java SDK
 
 [.text]
-The ESMF SDK helps you to build solutions, model your Aspect data, implement your own customized
-Aspect or want to integrate existing assets and systems within the Java programming language.
+Helps to build software solutions based on Aspect Models. Implement your own customized aspect services using Java.
```

Comment:
Create Aspect APIs for software solutions. The SDK supports by understanding Aspect Models and generating Java code & artefacts.

Reply:
Done with https://github.com/bci-oss/eclipse-esmf.github.io/pull/4/commits/ff3647f97983979796424d4a2f48778b81ceb294 ✔